### ISSUE TITLE
Nahlašování do SKC bez fyzického výtisku

### ISSUE THREAD

**Issue by BarkaHorakova** [digitalizace/digivpraxi#7]
Otevírám problematiku zasílání záznamů do SKC, když fyzicky dokument nemáme a půjčujeme si ho z jiné knihovny. Zatím to moc knihoven asi nedělalo. V Pardubicích jsme si nechali povolit duplicitní pole 910 a zasílali jsme záznam rovnou z našeho katalogu - vyplňovali jsme 910 pro PAG001 a druhou 910 pro knihovnu, ze které si dokument půjčujeme. Zárověň jsme dodali pole 911 pro PAG001 se statusem.
Zdálo se, že to funguje dobře. SKC využíváme i pro nahlašování do registru digitalizace. Ovšem ukázalo se, že tímto způsobem to nakonec možné není. Duplicitní 910 mohou mít jen knihovny, které samy o sobě fungují jako souborný katalog. Navíc je problém, že 910 identifikuje vlastníka dokumentu, kterým fakticky nejsme...
Pan Záhořík nám takové záznamy zakázal, ale nenavrhl žádné řešení (argument "mě to nezajímá, ať si to udělají, jak chtějí, ale takhle ne" mi moc konstruktivní nepřipadá :-)).
Jak to tedy správně dělat? Budu ráda za tipy a rady, protože níže popsaný způsob se mi nezdá úplně optimální.

- přes formulář k záznamu připíšeme knihovnu, ze které si půjčujeme
- u nás v katalogu vytvoříme záznam bez pole 910, jen s 911, odešleme do SKC
- nahlásíme do RD
- protože digitalizujeme službou, do našeho katalogu doplníme do 910 siglu knihovny, která nám dokument půjčila
- vygenerujeme XML pro digitalizační firmu
- smažeme 910 s cizí siglou a změníme status digitalizace
- odešleme do SKC

Je to složité, je tam spoustu prostoru pro chyby, ale dejme tomu.
Problém je s dokumenty, které jsou unikátní a žádný záznam v SKC, kam by se dalo připsat, neexistuje. Pak mě napadá jen to, že záznam odešleme z našeho katalogu s naší siglou bez signatury, pak nahlásíme přes formulář knihovnu, která dokument opravdu vlastní a nás přes formulář zase vymažeme. 

Poznámka na okraj - o ČNB i o ISSN můžete žádat, aniž byste byli vlastníky dokumentu..

**Comment by luckajirku** [comment]
No to jsi mě nepotěšila... Ať to máme provázané, doplním odkaz na issue z r. 2021, které jsem zakládala na standardech https://github.com/NLCR/Standard_NDK/issues/166 a taky na [dokument](https://docs.google.com/document/d/1rwctEAYbbX_QY-WRWJp84XckIl70eysHSbpKb8ZTgWo/edit), který vznikl k online schůzce 15. 3. 2024, která skončila s tím, že řešením je použití duplicitní 910, které pro SKC nepředstavuje problém. V dokumentu jsou popsány i postupy Hradce, Ostravy a Kladna. My jsme se teda od té schůzky posunuli, co se týká RD - máme upravený skript pro export marcxml z Alephu a teď jsme zpětně dohlašovali několik knihoven - vlastník předlohy tam je uveden jako vlastník, ale co se bib záznamu týká, odkaz vede do našeho Alephu na ten záznam po RK. Pořád to beru, že v RD není cílem lokalizovat předlohu, ale získat co nejpřesnější informace o titulu.
Každá knihovna to dělá trochu jinak, každá to v určitých chvílích ohýbá nebo dělá zbytečné opakované změny sigel, vyplňování a mazání atd. (každopádně Pardubice co do složitosti vedou, to se musí nechat:-)). Ale to není cesta - cílem mělo být ne najít další způsoby, jak to obcházet, ale najít cestu, jak ideálně s jedním záznamem projít celým procesem. A na začátku vzít jako fakt, že je v pořádku (a že je to přínosné), že krajská knihovna katalogizuje fond jiné knihovny z kraje do svého katalogu (a ten záznam tam zůstává pro tu digi verzi) - i proto, že se pak o tu verzi stará a zpřístupňuje ji ve své DK.
Nějaká cesta být musí. I když nejsme smart a musíme vystačit jen s přirozenou inteligencí:-). 
Každopádně teď je to zralé na další schůzku (kde by byl i p. Záhořík), když to odsouhlasené řešení nelze realizovat.

**Comment by BarkaHorakova** [comment]
Souhlasím, navíc tím, že pro digitalizaci uděláme kvalitní záznam, pozvedneme Souborný katalog :-). 
U nás je situace komplikovanější o to, že digitalizujeme službou a podle standardů je lokace neopakovatelný element, takže v záznamu musíme aspoň chvíli (než pro firmu vyexportujeme xml) držet cizí siglu a to na prvním místě...
A opravdu mi přijde důležité vytvořit nějaký hladký proces, který zohlední provázanost vlastního katalogu, SKC a registru. 
Teď jsem vyzkoušela, že záznam odejde a objeví se v SKC, i když má vyplněné jen pole 911 a 910 úplně chybí.
